project/libs/account/blog-user/src/blog-user-module/blog-user.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { BlogUserRepository } from './blog-user.repository';
+import { BlogUserFactory } from './blog-user.factory';
+
+@Module({
+  providers: [BlogUserFactory, BlogUserRepository],
+  exports:[BlogUserRepository]
+})
```

Comment:
Самая важная часть в модуле это service (там где бизнес логика).


```suggestion
@Module({
  providers: [BlogUserService, BlogUserFactory, BlogUserRepository],
  exports:[BlogUserService]
})
```